requirements.lock

```diff
@@ -3,50 +3,120 @@
 #
 # last locked with the following flags:
 #   pre: false
-#   features: ["cb7"]
-#   all-features: false
+#   features: []
+#   all-features: true
```

Comment:
I wouldn't enable `all-features` for the prod requirements, I'd keep it to just the "cb7" feature